src/monitor/axiom_client.py

```diff
@@ -1,6 +1,7 @@
 import os
 import requests
 import axiom
+import gevent
```

Comment:
why import 'gevent'?

Reply:
Forgot to delete it. At first, it was obtained with the carrent_Worker = Gevent.getcurrent (), and it could not be obtained later.